docs/v2-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 22, 2024
+
+### Timeplus Proton v1.5.14
+  * New feature: Remote user-defined functions (UDFs) are now supported.
```

Comment:
We support remote UDF long time ago. In 1.5.14, the user can create remote UDF via SQL (previously via REST API, or Timeplus Enterprise user creates it via UI)